admin_manual/installation/nginx-root.conf.sample

```diff
@@ -86,7 +86,7 @@ server {
     # only for Nextcloud like below:
     include mime.types;
     types {
-        application/javascript js mjs;
+        text/javascript js mjs;
```

Comment:
this seems to be the actual change? :+1:

Reply:
Yes but we need the other one as well to gzip compress it.